packages/block-editor/src/components/spacing-sizes-control/spacing-input-control.js

```diff
@@ -163,6 +163,8 @@ export default function SpacingInputControl( {
 		! showCustomValueControl &&
 		currentValueHint !== undefined;
 
+	const maxCustomValues = { px: 300, '%': 100, vw: 100, vh: 100 };
```

Comment:
This could be moved to file scope and named `MAX_CUSTOM_VALUES` to avoid the little bit of overhead of redefining it on every render.